From 37285c40e007642d66a5fbf47035e7820248221a Mon Sep 17 00:00:00 2001
From: Thomas Bourgoin <thomas.bourgoin@foss.st.com>
Date: Fri, 23 Feb 2024 14:17:58 +0100
Subject: [PATCH 0663/1695] plat-stm32mp1: conf: update flavorlist MP15-HUK-DT

Add not upstreamed board to the flavorlist MP15-HUK-DT.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 98ccff967b05fb6caedefbfe81f5597e222490d0 plat-stm32mp1: conf: update flavorlist MP15-HUK-DT]
Signed-off-by: Thomas Bourgoin <thomas.bourgoin@foss.st.com>
Change-Id: I9c113da899c5f1c782f8cefe58be34c7bd014f83
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/362483
Domain-Review: Gatien CHEVALLIER <gatien.chevallier@st.com>
Reviewed-by: Etienne CARRIERE <etienne.carriere@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
---
 core/arch/arm/plat-stm32mp1/conf.mk | 11 ++++++++++-
 1 file changed, 10 insertions(+), 1 deletion(-)

diff --git a/core/arch/arm/plat-stm32mp1/conf.mk b/core/arch/arm/plat-stm32mp1/conf.mk
index 98bb333fa..f94115512 100644
--- a/core/arch/arm/plat-stm32mp1/conf.mk
+++ b/core/arch/arm/plat-stm32mp1/conf.mk
@@ -45,9 +45,18 @@ flavorlist-1G = $(flavorlist-cryp-1G) \
 		$(flavorlist-no_cryp-1G)
 
 flavorlist-MP15-HUK-DT = $(flavor_dts_file-157A_DK1) \
+			 $(flavor_dts_file-157A_ED1) \
+			 $(flavor_dts_file-157A_EV1) \
 			 $(flavor_dts_file-157C_DK2) \
 			 $(flavor_dts_file-157C_ED1) \
-			 $(flavor_dts_file-157C_EV1)
+			 $(flavor_dts_file-157C_EV1) \
+			 $(flavor_dts_file-157D_DK1) \
+			 $(flavor_dts_file-157D_ED1) \
+			 $(flavor_dts_file-157D_EV1) \
+			 $(flavor_dts_file-157F_DK2) \
+			 $(flavor_dts_file-157F_ED1) \
+			 $(flavor_dts_file-157F_EV1)
+
 
 flavorlist-MP15 = $(flavor_dts_file-157A_DHCOR_AVENGER96) \
 		  $(flavor_dts_file-157A_DK1) \
-- 
2.47.2

